docs/src/test/java/docs/http/javadsl/HttpClientExampleDocTest.java

```diff
@@ -170,6 +186,28 @@ public void testSingleRequestExample() {
     //#single-request-example
   }
 
+  // compile only test
+  public void singleRequestInActorExample1() {
+    //#single-request-in-actor-example
+      class SingleRequestInActorExample extends AbstractActor {
```

Comment:
This has a bit weird indentation. If it should align properly with the imports, this line should have no indentation.